src/ColumnChart/components/ColumnChartContainer.ts

```diff
@@ -0,0 +1,7 @@
+import { SFC, createElement } from "react";
+import BarChartContainer, { BarChartContainerProps } from "../../BarChart/components/BarChartContainer";
+
+const ColumnChartContainer: SFC<BarChartContainerProps> = (props) =>
+    createElement(BarChartContainer, { ...props, orientation: "column" });
+
+export { ColumnChartContainer as default };
```

Comment:
Inline this. 
export default const.........

Reply:
Ahh, forgot... inlined default exports can only be done with classes.